includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -174,14 +174,25 @@ protected function get_body_node() {
 	 * @return float|int|string Returns a numeric dimension value, or an empty string.
 	 */
 	public function sanitize_dimension( $value, $dimension ) {
-		if ( empty( $value ) ) {
+
+		/**
```

Comment:
Minor thing, but this should be just:

```php
// Allows 0 as valid dimension.
```

Same for the comment below.

The `/**` comments are for phpdoc specifically, and there is no need for multi-line here.